evalml/tests/model_understanding_tests/test_graphs.py

```diff
@@ -1418,3 +1422,48 @@ def test_visualize_decision_trees(fitted_tree_estimators, tmpdir):
     src = visualize_decision_tree(estimator=est_reg, filled=True, max_depth=2)
     assert src.format == 'pdf'
     assert isinstance(src, graphviz.Source)
+
+
+def test_linear_coefficients_errors():
+    X = pd.DataFrame([[1, 2, 3, 5],
+                      [3, 5, 2, 1],
+                      [5, 2, 2, 2],
+                      [3, 2, 3, 3]])
+    y = pd.Series([2, 1, 3, 4])
+
+    dt = DecisionTreeRegressor()
+    dt.fit(X, y)
+
+    with pytest.raises(ValueError, match="Linear coefficients are only available for linear family models"):
+        get_linear_coefficients(dt)
+
+    lin = LinearRegressor()
+
+    with pytest.raises(ValueError, match="This linear estimator is not fitted yet."):
+        get_linear_coefficients(lin)
+
+
+def test_linear_coefficients_output():
```

Comment:
If I'm understanding correctly, this test uses two different linear models and checks that the method works. I think we could parametrize this test to use go through all of the linear model estimators and save duplication of code (no need to write the same asserts multiple times for different estimators)